setup/azurecmdfiles.wxi

```diff
@@ -578,6 +578,15 @@
                         <Component Id="cmpF08FE4050A937C6DD895E92ABAE2A275" Guid="*">
```

Comment:
I see no changes in your wix, and changes for other RPs.  Could you make sure to pull from dev, clean your repo, rebuild, and regen the installer?